tests/phpunit/CRM/Contribute/Form/Task/StatusTest.php

```diff
@@ -45,6 +98,7 @@ public function testUpdatePendingContribution() {
     $form->buildQuickForm();
 
     $params = [
+      "is_email_receipt" => '1',
```

Comment:
Thanks for the test just a couple notes. This test is the one where you want to test when the checkbox is unchecked, so you don't want this line. In real life it would be missing from the params when the checkbox is unchecked.

Reply:
Ah you are right